deployment/keystone/changeset/internal/update_don.go

```diff
@@ -89,24 +91,37 @@ func UpdateDon(lggr logger.Logger, req *UpdateDonRequest) (*UpdateDonResponse, e
 		return nil, fmt.Errorf("failed to compute configs: %w", err)
 	}
 
-	_, err = AppendNodeCapabilitiesImpl(lggr, req.appendNodeCapabilitiesRequest())
```

Comment:
removed from here so that the top level changeset can leverage existing AppendNodeCapabilities changeset

enable reuse of public api